riscv/tests/riscv_data/evm/src/lib.rs

```diff
@@ -24,17 +19,18 @@ fn main() {
         b256!("e3c84e69bac71c159b2ff0d62b9a5c231887a809a96cb4a262a4b96ed78a1db2");
     let mut db = CacheDB::new(EmptyDB::default());
 
+    let bytecode_len = get_prover_input(0);
+    let bytecode: Vec<_> = (1..(bytecode_len + 1)).map(|idx| get_prover_input(idx) as u8).collect();
+
     // Fill database:
-    let bytecode = Bytes::from_static(BYTECODE);
+    let bytecode = Bytes::from(bytecode);
```

Comment:
Does that work with `Vec<usize>` as well? Then you need to call `get_prover_input()` 4x fewer times. But if bytecodes are short, this is probably fine.

Reply:
Yea for now they're short so I just wanted to keep it simple. But yea makes sense to do that eventually